src/components/Typography/CardTitle.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components'
+import is from 'styled-is'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+
+import modifiers from './modifiers'
+
+const CardTitle = styled.h3`
+  color: ${props => props.theme.colors.text};
+  font-weight: 500;
+  font-size: ${remcalc(21)};
+  line-height: ${remcalc(24)};
+  padding: ${remcalc(11)} 0 ${remcalc(13)} 0;
+  ${modifiers}
+
+  ${is('regular')`
+    font-weight: 400;
+  `};
+
+  ${breakpoint('tablet')`
+    font-size: ${remcalc(26)};
+    line-height: ${remcalc(30)};
+    padding: ${remcalc(12)} 0 ${remcalc(12)} 0;
+    ${modifiers}
+
+    ${is('bigger')`
```

Comment:
Engineers, please note that this prop is only used in the "related projects" section on the SEO template & is likely to be an unintended design discrepancy.

Spoken to Antonas about this. Should get ironed out once he picks up [the trello ticket for that component](https://trello.com/c/9uQ3q3Fk/306-casestudycard-component).

So I've also added a note about (potentially) removing this prop, to the checklist on that trello ticket.

Hope that's clear enough?